File: fizz-buzz.js
function main() {
  const N = 15;
  for (const n of [...Array(N)].map((_, i) => i + 1)) (!(n % 3)) && (!(n % 5)) ? console.log('FizzBuzz') : (!(n % 5)) ? console.log('Buzz') : (!(n % 3)) ? console.log('Fizz') : console.log(n);
}

main()

File: cpp-code/fizz-buzz.cpp
#include <iostream>
using namespace std;

int main() {
  int n;
  cin >> n;

  for (int i = 1; i <= n; ++i) {
    i % 5 == 0 && i % 3 == 0 ? cout << "FizzBuzz" << endl :
    i % 5 == 0 ? cout << "Buzz" << endl :
    i % 3 == 0 ? cout << "Fizz" << endl :
    cout << i << endl;
  }
}
